snippets/scripts/components-forge-field-to-compass-components/migrateCompassCFToComponent.py

```diff
@@ -284,56 +297,45 @@ def main():
     )
     issue_count = issues["total"]
 
-    if issue_count > 0:
+    if issue_count <= 0:
         print(
-            f"Successfully retrieved {issue_count} issues that have a Compass custom field "
-            f"value.\n"
+            f"The project: {PROJECT_KEY} doesn't have issues related to Compass custom fields."
         )
-        formatted_issues_dict = get_formatted_issues(
-            issues["issues"], compass_custom_field_fieldid
+        quit()
+    else:
+        print(
+            f"Successfully retrieved {issue_count} issues that have a Compass custom field value.\n"
         )
-        # Loop through pagination requests to get all issues
         number_loop = issue_count // max_results
-        for i in range(0, number_loop):
-            start_at = (i + 1) * max_results
```

Comment:
😱 Does this mean the script has always been skipping the first `max_results` issues?

Reply:
No I did not skip the first loop. If you see preivous line 292, I store first group in the all issues queue first. 

My current change will take out the all issues list. I just store local issues and migrate group of issues immediately.  Feel free to let me know if there is any other issues.


![Screenshot 2024-06-11 at 14 25 00](https://github.com/atlassian-labs/compass-examples/assets/143025715/3888879b-e015-4700-829f-68e7be6af7c2)

Reply:
Ah right, that rings some bells. 👍

Reply:
I know so I try to make current loop looks more resonable,